system_strings/sysmes_text.en

```diff
@@ -735,7 +735,7 @@ Chasing after Arcueid.
 To the city at night.
 Back to her apartment.
 ...To a downtown back alley.
-Seeing life in midst of death.
+Seeing life in the midst of death.
```

Comment:
that's 34 characters

Reply:
![image](https://user-images.githubusercontent.com/90473809/180940931-b46f8754-f47d-40d6-baa1-4f43186ef541.png)
removing that period would work though

Reply:
Ah, I'd seen it on the save screen where there's a bunch more space. I'll just remove the period.